documents/customer.md

```diff
@@ -145,6 +145,248 @@ client.customer.fetch(customerId)
 
 -------------------------------------------------------------------------------------------------------
 
+### Add Bank Account of Customer
+
+```py
+customerId = "cust_N5mywh91sXB69O"
+
+client.customer.addBankAccount(customerId, {
+    "ifsc_code" : "UTIB0000194",
+    "account_number"         :"916010082985661",
+    "beneficiary_name"      : "Pratheek",
+    "beneficiary_address1"  : "address 1",
+    "beneficiary_address2"  : "address 2",
+    "beneficiary_address3"  : "address 3",
+    "beneficiary_address4"  : "address 4",
+    "beneficiary_email"     : "random@email.com",
+    "beneficiary_mobile"    : "8762489310",
+    "beneficiary_city"      :"Bangalore",
+    "beneficiary_state"     : "KA",
+    "beneficiary_country"   : "IN"
+})
+```
+
+**Parameters:**
+
+| Name          | Type        | Description                                 |
+|---------------|-------------|---------------------------------------------|
+| customerId*  | string      | Unique identifier of the customer.  |
+| account_number | string      | Customer's bank account number.  |
+| beneficiary_name | string  | The name of the beneficiary associated with the bank account.  |
+| beneficiary_address1 | string      | The virtual payment address.  |
```

Comment:
Is this correct? should it be VPA, VPA is used for upi right?

Reply:
I mentioned the description as it is showing in the doc, should i have to change something here? 

<img width="690" alt="Screenshot 2024-03-11 at 8 01 27 PM" src="https://github.com/razorpay/razorpay-python/assets/89454448/057d4e02-2bd3-4259-a10f-3d7228ff3845">